Date and Time Series tutorial update (#500)
plotly/plotly.github.io: _posts/fundamentals/2015-04-24-dates-time-series-and-timestamp.md

 - title: Entering Dates and Times in Chart Studio’s Grid
   sub-steps:
    - copy: "To enter dates directly in the grid, you’ll need to use the format: yyyy-mm-dd HH:MM:SS.ssssss. Note that the hour must be a number between 00 and 23, with hours 12 through 23 reserved for PM. Your entry must include the four-digit year."
      img: "![dateandtime](../static/images/dates-time-series-and-timestamp/dates and times.png)"
    - copy: "You can truncate the fractional seconds to any number of digits. All other truncations require you to keep entire fields. Here are a few common mistakes to avoid."
      img: "![fractional seconds](../static/images/dates-time-series-and-timestamp/fractional seconds.png)"

 - title: Importing Files with Dates and Times
   sub-steps:
    - copy: "[Uploading a data set](https://help.plot.ly/add-data-to-the-plotly-grid/#step-2-option-2-import-a-data-file) into the grid from your computer, or an online repository? Chart Studio will parse the data for you into our native format so you’re ready to customize your graph."
      img: "![Date format and time series](../static/images/dates-time-series-and-timestamp/upload.png)"

 - title: Custom Format
   sub-steps:

>     - copy: "When you make a graph in Chart Studio with dates and times, the default labelling picks spacing that fits the graph. Here the dates is included year, month, and day, but Chart Studio displays month and year by default."

Reviewer: ```suggestion
    - copy: "When you make a graph in Chart Studio with dates and times, the default labelling picks spacing that fits the graph. Here the dates included year, month, and day, but Chart Studio displays month and year by default."
```

      img: "![Date format and time series](../static/images/dates-time-series-and-timestamp/dateFormat.png)"

>     - copy: "To customize date format, from 'Axes' under 'Style' menu choose 'Tick Labels' submenu. Next, select the axis you wish to modify, and then set 'advanced(d3-time-format)' for 'Label Format' attribute. A text box would be appeared for which you can choose a [custom time format](https://github.com/d3/d3-time-format/blob/v2.1.3/README.md#timeFormat)."

Reviewer: ```suggestion
    - copy: "To customize date format, from 'Axes' under 'Style' menu choose 'Tick Labels' submenu. Next, select the axis you wish to modify, and then set 'advanced(d3-time-format)' for 'Label Format' attribute. A text box will appear where you can enter a [custom time format](https://github.com/d3/d3-time-format/blob/v2.1.3/README.md#timeFormat)."
```
